patch/kernel/archive/sunxi-5.15/0010-allwinner-add-sunxi_get_soc_chipid-and-sunxi_get_ser.patch

```diff
@@ -0,0 +1,110 @@
+From a27f390d577b4e52da959afb84e5280f7d71dd0e Mon Sep 17 00:00:00 2001
+From: afaulkner420 <afaulkner420@gmail.com>
+Date: Fri, 25 Mar 2022 20:38:14 +0000
+Subject: [PATCH 10/11] allwinner: Add sunxi_get_soc_chipid() and
+ sunxi_get_serial function
+
+---
+ drivers/nvmem/sunxi_sid.c     | 28 +++++++++++++++++++++++
+ drivers/nvmem/sunxi_sid.c.rej | 43 +++++++++++++++++++++++++++++++++++
```

Comment:
.rej is probably not needed.

Reply:
I didn't notice that was still there. I'll remove...